opengever/kub/client.py

```diff
@@ -117,6 +117,20 @@ def create_person(self, data):
             logger.exception('Creating person failed')
             return None
 
+    def create_organization(self, data):
+        url = '{}organizations'.format(self.kub_api_url)
+        try:
+            resp = self.session.post(url, json=data)
+            resp.raise_for_status()
+        except requests.RequestException:
+            logger.exception('Creating organization failed: {}'.format(resp.content))
```

Comment:
You should not use `str.format` or f-strings for logging but instead printf style with positional parameters.
The advantages are:
- It's only evaluated if the required logging level is reached
- In log aggregation tools like Sentry you get only one event for all combinations of parameter values. With your approach you get a new (different) event for every new parameter value.

```suggestion
            logger.exception('Creating organization failed: %s', resp.content)
```